packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -3,58 +3,276 @@
+/// corner and so ~76px x ~38px would be used to render both corners on a given
+/// side.
+///
+/// This shape will always have 4 linear edges and 4 90º curves. However, for
+/// rectangles with small values of width or height (ie.  <20 lpx) and a low
+/// aspect ratio (ie. <0.3), the rendered shape will appear to have just 2
+/// linear edges and 2 180º curves.
+///
+/// The example below shows how to render a continuous rectangle on screen.
 ///
 /// {@tool sample}
 /// ```dart
 /// Widget build(BuildContext context) {
-///   return Material(
-///     shape: ContinuousRectangleBorder(
-///       borderRadius: BorderRadius.circular(28.0),
+///   return Container(
+///     alignment: Alignment.center,
+///     child: Material(
+///       color: Colors.blueAccent[400],
+///       shape: const ContinuousRectangleBorder(cornerRadius: 75.0),
+///       child: const SizedBox(
+///         height: 200.0,
+///         width: 200.0,
+///       ),
 ///     ),
 ///   );
 /// }
 /// ```
 /// {@end-tool}
 ///
+/// The image below depicts a [ContinuousRectangleBorder] with a width
+/// and height of 200 and a curve radius of 75.
+///
+/// ![](https://flutter.github.io/assets-for-api-docs/assets/painting/Shape.continuous_rectangle.png)
+///
 /// See also:
 ///
-/// * [RoundedRectangleBorder] Which creates rectangles with rounded corners,
-///   however its straight sides change into a rounded corner with a circular
-///   radius in a step function instead of gradually like the
-///   [ContinuousRectangleBorder].
+/// * [RoundedRectangleBorder], which is a rectangle whose corners are
+///   precisely quarter circles.
+/// * [ContinuousStadiumBorder], which is a stadium whose two edges have a
+///   continuous transition into its two 180º curves.
+/// * [StadiumBorder], which is a rectangle with semi-circles on two parallel
+///   edges.
 class ContinuousRectangleBorder extends ShapeBorder {
-  /// The arguments must not be null.
+  /// Creates a continuous cornered rectangle border.
+  ///
+  /// The [side] and [cornerRadius] arguments must not be null.
   const ContinuousRectangleBorder({
     this.side = BorderSide.none,
-    this.borderRadius = BorderRadius.zero,
+    this.cornerRadius = 0.0,
   }) : assert(side != null),
-       assert(borderRadius != null);
+       assert(cornerRadius != null);
 
   /// The radius for each corner.
   ///
-  /// Negative radius values are clamped to 0.0 by [getInnerPath] and
-  /// [getOuterPath].
-  final BorderRadiusGeometry borderRadius;
+  /// The radius will be clamped to 0 if a value less than 0 is entered as the
+  /// radius. By default the radius is 0. This value must not be null.
+  ///
+  /// Unlike [RoundedRectangleBorder], there is only a single radius value used
+  /// to describe the radius for every corner.
+  final double cornerRadius;
 
   /// The style of this border.
+  ///
+  /// If the border side width is larger than 1/10 the length of the smallest
```

Comment:
How is this number derived?

Reply:
This is a rough approximation of where I feel the visual difference would be imperceptible.